src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRefVCF.kt

```diff
@@ -7,23 +7,18 @@ import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.validate
-import com.google.common.collect.HashMultimap
-import com.google.common.collect.Multimap
-import com.google.common.collect.Range
 import htsjdk.variant.variantcontext.VariantContext
 import htsjdk.variant.vcf.VCFAltHeaderLine
 import htsjdk.variant.vcf.VCFHeaderLine
 import htsjdk.variant.vcf.VCFHeaderVersion
 import net.maizegenetics.phgv2.utils.*
-import org.apache.logging.log4j.LogManager
-import java.io.File
 import java.util.*
 import java.util.logging.Logger
 
 
-class BuildRefVcf : CliktCommand() {
+class CreateRefVCF : CliktCommand() {
```

Comment:
I would check to see if this should "Vcf" instead of "VCF". Clikt might convert this to `create-ref-v-c-f` based on its translation logic

Reply:
agreed - clikt does convert it correctly to create-ref-vcf, but I'm going to change to CreateRefVcf for consistency.